src/main/java/uk/gov/digital/ho/hocs/casework/migration/api/MigrationCaseDataService.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.digital.ho.hocs.casework.migration.api;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.casework.api.dto.CaseDataType;
+import uk.gov.digital.ho.hocs.casework.client.auditclient.AuditClient;
+import uk.gov.digital.ho.hocs.casework.client.infoclient.InfoClient;
+import uk.gov.digital.ho.hocs.casework.domain.exception.ApplicationExceptions;
+import uk.gov.digital.ho.hocs.casework.domain.model.CaseData;
+import uk.gov.digital.ho.hocs.casework.domain.repository.CaseDataRepository;
+
+import java.time.LocalDate;
+import java.util.Map;
+import java.util.UUID;
+
+import static net.logstash.logback.argument.StructuredArguments.value;
+import static uk.gov.digital.ho.hocs.casework.application.LogEvent.*;
```

Comment:
Bringing in the entire of this package - we should prefer individual imports.

Reply:
Fixed.